src/screens/TopicMemberScreen/index.js

```diff
@@ -109,6 +109,8 @@ const TopicMemberScreen = () => {
         setTopicDataUnfollowedUsers(newDataUnfollowed);
       } else {
         setInitialMember([...newDataFollowed, ...newDataUnfollowed]);
+        setTopicDataFollowedUsers(newDataFollowed);
+        setTopicDataUnfollowedUsers(newDataUnfollowed);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The state setters `setTopicDataFollowedUsers` and `setTopicDataUnfollowedUsers` are now being called in an additional context. Ensure that this doesn't introduce any unintended side effects, especially considering asynchronous state updates.
```diff
+ setTopicDataFollowedUsers(newDataFollowed);
+ setTopicDataUnfollowedUsers(newDataUnfollowed);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->